05. 효율성(투포인터 알고리즘, 슬라이디윈도우, 해쉬)/06. 학급 회장(해쉬)/김보경/index.js

```diff
@@ -1,8 +1,25 @@
 function solution(s) {
   let answer;
+  let countResults = {};
+
+  [...s].forEach((el) => {
+    if (el in countResults) {
+      countResults[el]++;
+    } else {
+      countResults[el] = 1;
+    }
+  });
```

Comment:
reduce로도 할 수 있을 거 같아요~

```suggestion
  const countResults = [...s].reduce((acc, cur) => {
   acc[cur] = (acc[cur] || 0) + 1;
   return acc;
  }, {});
```

Reply:
오... reduce를 이런식으로 활용 가능하군요 아영님 덕분에 하나 알아갑니다 굿굿